pkg/repo/index_test.go

```diff
@@ -131,7 +131,7 @@ func TestMerge(t *testing.T) {
 
 	if len(ind1.Entries) != 2 {
 		t.Errorf("Expected 2 entries, got %d", len(ind1.Entries))
-		vs := ind1.Entries["dreadnaught"]
+		vs := ind1.Entries["dreadnought"]
```

Comment:
afaict, this code is unreachable/untested/would not work.

See lines 116/122.

I could be wrong, but as w/ `jabbathehut`, if there's a specific reason for misspelling something, it should be clearly identified.